types/react-native-dotenv.d.ts

```diff
@@ -6,4 +6,5 @@ declare module '@env' {
   export const MAPBOX_API_TOKEN: string;
   export const MAPBOX_STOP_PLACES_STYLE_URL: string;
   export const AUTHORITY_ID: string;
+  export const APP_ORG: string;
```

Comment:
Burde det være en import av `AppOrgs` lengre opp her, og så statisk type `APP_ORG` her?
```suggestion
  export const APP_ORG: AppOrgs;
```